app/models/item.rb

```diff
@@ -35,4 +35,9 @@ def self.least_popular
   def can_be_fulfilled(amount_wanted)
     amount_wanted < inventory
   end
+
+  def default_photo?
+    self.image = "https://dapp.dblog.org/img/default.jpg" if self.image == ""
```

Comment:
Interesting choice to have a default photo.

Reply:
I think it was mentioned in a user story, if you don't fill one in then you are given a default

Reply:
There was a line in the controller as well that sets a default image if the image is "" just double check that they don't conflict.